src/pages/Posts.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import './Post.css';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import { toast } from 'react-hot-toast'
+
+const Posts = () => {
+    const [posts, setPosts] = useState([]);
+    const [comments, setComments] = useState([]);
+    const [newComment, setNewComment] = useState('');
+    const handleCommentSubmit = async (id) => {
+        setComments((prevComments) => [...prevComments, newComment]);
+        console.log(newComment, id)
+        try {
+            const response = await axios.post(`/posts/:${id}/comment`, { comment: newComment });
+            console.log(response.data)
```

Comment:
if you are not using the response aside from a console log, it is somewhat redundant